### PR TITLE
[hi] Localize container-image.md

### PR DESCRIPTION
**Pull request by Garima-Negi**
Related to issue : [https://github.com/cncf/glossary/issues/270](https://github.com/cncf/glossary/issues/270)

### CODE REVIEW

File: content/hi/container-image.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+title: कंटेनर इमेज
+status: Feedback Appreciated
+category: अवधारणा
+---
+
+एक कंटेनर इमेज (छवि) एक अपरिवर्तनीय, स्थिर फ़ाइल है जिसमें एक कंटेनर के निर्माण के लिए निर्भरताएँ होती हैं। इन निर्भरताओं में एकल निष्पादन योग्य बाइनरी फ़ाइल, सिस्टम लाइब्रेरी, सिस्टम उपकरण, एनवायरनमेंट वेरिएबल्स और अन्य आवश्यक प्लेटफ़ॉर्म समायोजन शामिल हो सकते हैं। कंटेनर इमेजिस एक एप्लिकेशन के कंटेनरीकरण से उत्पन्न होती हैं और आमतौर पर कंटेनर रजिस्ट्रियों में संग्रहीत की जाती हैं, जहां उन्हें कंटेनर रनटाइम इंटरफ़ेस (CRI) का उपयोग करके एक अलग प्रक्रिया के रूप में डाउनलोड और चलाया जा सकता है। एक कंटेनर इमेज ढांचे को ओपन कंटेनर इनिशिएटिव (OCI) द्वारा परिभाषित मानक स्कीमा का पालन करना चाहिए।
```
Comment (anubha-v-ardhan) [inline content/hi/container-image.md:7]:
```suggestion
एक कंटेनर इमेज (छवि) एक अपरिवर्तनीय, स्टैटिक फ़ाइल है जिसमें एक कंटेनर के निर्माण के लिए निर्भरताएँ होती हैं। इन निर्भरताओं में एक एक्सेक्युट करने योग्य बाइनरी फ़ाइल, सिस्टम लाइब्रेरी, सिस्टम उपकरण, एनवायरनमेंट वेरिएबल्स और अन्य आवश्यक प्लेटफ़ॉर्म समायोजन शामिल हो सकते हैं। कंटेनर इमेज एक एप्लिकेशन के कंटेनरीकरण से उत्पन्न होती हैं और आमतौर पर कंटेनर रजिस्ट्रियों में संग्रहीत की जाती हैं, जहां उन्हें कंटेनर रनटाइम इंटरफ़ेस (CRI) का उपयोग करके एक अलग प्रक्रिया के रूप में डाउनलोड और चलाया जा सकता है। एक कंटेनर इमेज ढांचे को ओपन कंटेनर इनिशिएटिव (OCI) द्वारा परिभाषित मानक स्कीमा का पालन करना चाहिए।
```

Comment (anubha-v-ardhan) [reply content/hi/container-image.md:7]: I think we should not translate "static" and "execute" as they being technical terms are always used in their English version and may get difficult to understand what we are saying upon translation

File: content/hi/container-image.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+परंपरागत रूप से, एप्लिकेशन सर्वर प्रति वातावरण कॉन्फ़िगर किए जाते हैं, और एप्लिकेशन उन पर तैनात किए जाते हैं। वातावरणो के बीच कोई भी गलत कॉन्फ़िगरेशन समस्याग्रस्त है और अक्सर डाउनटाइम (स्र्कावट) या असफल परिनियोजन की ओर ले जाता है। एक एप्लिकेशन का वातावरण दोहराने योग्य और अच्छी तरह से परिभाषित होना चाहिए; अन्यथा, वातावरण से संबंधित बग (प्रोग्राम में दोष) की संभावना बढ़ जाती है। जब अनुप्रयोग वातावरण अपरिभाषित या गलत होते हैं, तो अनुप्रयोगों का क्षैतिज और लंबवत स्केलिंग चुनौतीपूर्ण हो जाता है।
```
Comment (anubha-v-ardhan) [inline content/hi/container-image.md:9]:
```suggestion
परंपरागत रूप से, एप्लिकेशन सर्वर प्रति वातावरण कॉन्फ़िगर किए जाते हैं, और एप्लिकेशन उन पर डेप्लॉय किए जाते हैं। वातावरणों के बीच कोई भी गलत कॉन्फ़िगरेशन समस्यात्मक हो सकता है और अक्सर डाउनटाइम (स्र्कावट) या असफल परिनियोजन की ओर ले जाता है। एक एप्लिकेशन का वातावरण दोहराने योग्य और अच्छी तरह से परिभाषित होना चाहिए; अन्यथा, वातावरण से संबंधित बग (प्रोग्राम में दोष) की संभावना बढ़ जाती है। जब एप्लिकेशन वातावरण अपरिभाषित या गलत होते हैं, तो एप्लिकेशनों का क्षैतिज और लंबवत स्केलिंग चुनौतीपूर्ण हो जाता है। 
```

File: content/hi/container-image.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+परंपरागत रूप से, एप्लिकेशन सर्वर प्रति वातावरण कॉन्फ़िगर किए जाते हैं, और एप्लिकेशन उन पर तैनात किए जाते हैं। वातावरणो के बीच कोई भी गलत कॉन्फ़िगरेशन समस्याग्रस्त है और अक्सर डाउनटाइम (स्र्कावट) या असफल परिनियोजन की ओर ले जाता है। एक एप्लिकेशन का वातावरण दोहराने योग्य और अच्छी तरह से परिभाषित होना चाहिए; अन्यथा, वातावरण से संबंधित बग (प्रोग्राम में दोष) की संभावना बढ़ जाती है। जब अनुप्रयोग वातावरण अपरिभाषित या गलत होते हैं, तो अनुप्रयोगों का क्षैतिज और लंबवत स्केलिंग चुनौतीपूर्ण हो जाता है। 
+
+कंटेनर इमेजिस किसी एप्लिकेशन को उसकी किसी भी रनटाइम निर्भरता के साथ बंडल करती हैं, जैसे कि एप्लिकेशन सर्वर। यह एक डेवलपर की मशीन सहित सभी वातावरणो में एकरूपता प्रदान करता है। कंटेनर इमेजिस का उपयोग अधिक से अधिक स्केलेबिलिटी की अनुमति देते हुए, आवश्यकतानुसार कई कंटेनरों को तत्काल अवतीर्ण करने के लिए किया जा सकता है।
```
Comment (anubha-v-ardhan) [inline content/hi/container-image.md:11]:
```suggestion
कंटेनर इमेजें किसी एप्लिकेशन को उसकी किसी भी रनटाइम निर्भरता, जैसे कि एप्लिकेशन सर्वर के साथ बंडल करती हैं। यह एक डेवलपर की मशीन सहित सभी वातावरणो में एकरूपता प्रदान करता है। कंटेनर इमेजों का उपयोग अधिक से अधिक स्केलेबिलिटी की अनुमति देते हुए, आवश्यकतानुसार कई कंटेनरों को तत्काल अवतीर्ण करने के लिए किया जा सकता है।
```

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]
Please have a look on these suggestions and commit them if they look good to you. If you find any inaccuracy or feel like a change is unecessary, free feel to discuss

**Comment by anubha-v-ardhan** [comment]
@Garima-Negi If time allows, please have a look at the suggestions

File: content/hi/container-image.md (hunk 2 above)
Comment (Garima-Negi) [reply content/hi/container-image.md:9]: 👍

File: content/hi/container-image.md (hunk 1 above)
Comment (Garima-Negi) [reply content/hi/container-image.md:7]: Completely agree! Will also add these to the spreadsheet. Thank you !

**Review by anubha-v-ardhan — APPROVED** [review]
LGTM!
Thank you @Garima-Negi :)

**Pull request merged**